deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset.yaml

```diff
@@ -25,11 +25,23 @@ spec:
       serviceAccountName: elastic-agent-standalone
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
+      #initContainers:
```

Comment:
should the following be uncommented for hints? or any other reason?

Reply:
Missed to add the explanation yeah...Thanks!